insights/parsers/tests/test_swift_conf.py

```diff
@@ -106,3 +129,11 @@ def test_object_expirer_conf():
     assert 'object-expirer' in result
     assert result.get('filter:cache', 'memcache_servers') == '172.16.64.60:11211'
     assert result.getint('object-expirer', 'report_interval') == 300
+
+
+def test_swift_conf():
+    swift_conf = SwiftConf(context_wrap(SWIFT_CONF))
+    assert 'swift-hash' in swift_conf.sections()
+    assert swift_conf.has_option('storage-policy:2', 'policy_type') is True
+    assert swift_conf.get('storage-policy:2', 'policy_type') == 'erasure_coding'
+    assert swift_conf.get('storage-policy:2', 'ec_type') == 'liberasurecode_rs_vand'
```

Comment:
@psachin please add tests for the examples section of the parser docstring.  See [this reference](https://insights-core-tutorials.readthedocs.io/en/latest/customtut_parsers.html#parser-testing) for more info.

Reply:
Sure